scapy/contrib/modbus.py

```diff
@@ -14,10 +14,7 @@
 # You should have received a copy of the GNU General Public License
 # along with Scapy. If not, see <http://www.gnu.org/licenses/>.
 
-# scapy.contrib.description = ModBus Protocol
-# scapy.contrib.status = loads
```

Comment:
Hey, I've spent some time to make hose comments right :/
Could you leave them ?